arcgis-js-api/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for ArcGIS API for JavaScript v3.18
-// Project: http://js.arcgis.com
-// Definitions by: Esri <http://www.esri.com>
+// Type definitions for ArcGIS API for JavaScript 3.19
+// Project: https://developers.arcgis.com/javascript/3/
+// Definitions by: Esri <https://github.com/Esri>
```

Comment:
I meant vs. a user account. Although you could of course tell me if you get a notification when I mention @Esri.